### PR TITLE
Custom events

### PR DESCRIPTION
**Pull request by halenander**
This change allows for custom events to be created. The new function is in util.go and the function call is in AddPods in cluster_utils.go. To test it, simply try to add a bad pod to the cluster. Then open the dashboard and go to events. A YAML file should also show up in the project's util folder.

Please check to make sure the alpine test pod works with these changes. I have been having some trouble with it, but it could be an unrelated issue.

### CODE REVIEW

**Comment by halenander** [comment]
related to #10

**Comment by JacksonCrawford** [comment]
On line 264 in `cluster_utils.go` the `NonfatalErrorCheck` is redundant because you already handle the error with your event writer.

File: pkg/util/util.go [hunk 1]
```diff
@@ -107,46 +113,81 @@ func InjectYamlCA(target, template, injectable string) {
...
+		eventType = "Warning"
+	} else {
+		eventReason = "Pod accepted"
+		eventType = "Normal"
+	}
+
+	// set a name that will change even for duplicate pods
+	eventName := podName + FormatTime()
+
+	//event := corev1.Event{
```
Comment (JacksonCrawford) [inline pkg/util/util.go:139]: May you please get rid of this unused Event?

**Review by frienderman2 — APPROVED** [review]

**Review by JacksonCrawford — APPROVED** [review]
lgtm

**Pull request merged**